x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -211,6 +214,49 @@ export const signalRulesAlertType = ({
           await ruleStatusService.error(errorString);
           wroteStatus = true;
         }
+        const timestampFieldCaps = await services.scopedClusterClient.fieldCaps({
+          index,
+          fields: timestampFields,
+          allow_no_indices: false,
+          include_unmapped: true,
```

Comment:
Ha! This API has 🐍 ... Interesting to see. I think the tide is changing from the deep core!